### PR TITLE
Adds multiple server entry handling

### PR DESCRIPTION
**Pull request by delambo**
- adds support for server webpack configuration with multiple server entries
- adds validation for server entry since we assume that there is at least one server entry named "main"

This pull request does not include any breaking changes to the api.

### CODE REVIEW

File: utils/__mocks__/webpackCompiler.js [hunk 1]
```diff
@@ -1,13 +1,37 @@
...
     path: 'fakePath',
     publicPath: 'publicPath',
   },
 };
+
+const multiEntryOptions = {
+  entry: {
+    main: ['main.js'],
+    additional: ['more.js'],
+  },
```
Comment (tizmagik) [inline utils/__mocks__/webpackCompiler.js]: Note that an entry can also be a string literal. The test would fail in that case.

Comment (delambo) [inline utils/__mocks__/webpackCompiler.js]: I think this works ok since it only uses the key names.

Comment (tizmagik) [inline utils/__mocks__/webpackCompiler.js]: Ah, I see. Maybe adding the test case with a string literal would be useful though.

**Comment by tizmagik** [comment]
@delambo Does this just add 'watcher' support for multiple entries? Because I think the current kyt on master supports multiple entries fine, no?

**Comment by delambo** [comment]
Yeah, sorry, I should have clarified. kyt/Webpack handles multiple entry points. What this pull request does is makes sure that kyt watches multiple entries during development.

File: utils/__mocks__/webpackCompiler.js (hunk 1 above)
Comment (delambo) [inline utils/__mocks__/webpackCompiler.js]: Added an additional mock and test for string literals.

**Comment by tizmagik** [comment]
Gotcha, sounds good!

**Pull request merged**